### ISSUE TITLE
Update translation feature and documentation for wcag 2.1 and newer

### ISSUE THREAD

**Issue by rvantonisse** [w3c/wcag-em-report-tool#333]
There needs to be an update of `docs/translation.md` to reflect how to create translations for newer versions of the [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/).

Current documentation applies to wcag version 2.0 only in combination with previous version of this tool.

The tool itself should host the scripts needed to create and update translations so that this can be updated in one place independent of the project's current maintainer.

### Update translation feature:

- [x] Add fallback to default language functionality

- [ ] Identify criteria by their section numbers instead, htmlIds may change per version of WCAG 2.x section numbering is expected to stay intact.

- [ ] Find a way to include the wcag specification into the translation service to have one approach to translate the application. (Pre render translation objects?)

- [ ] Update Dutch (nl) translation

### Translation feature fixes:

- [ ] Fix translation switching in scope view; Translation of Version and Level options are not actual. ($scope.$apply?)

### Update translation documentation:

- [ ] Add instructions for translations when a W3C Recommended translation of the WCAG is missing.

**Comment by hidde** [comment]
This is no longer relevant and superseded by #493
